src/content/en/2020/page-weight.md

```diff
@@ -23,9 +23,9 @@ featured_stat_label_3: The median number of mobile page requests
 
-## #PageWeightStillMatters
+### PageWeightStillMatters
 
-#PageWeightStillMatters would almost imply that it didn't or ever mattered. It might not have mattered when text based Craigslist launched. But 25 years ago when it was founded, Mosaic 1.0 also launched the same year, and Waterfalls by TLC was a top hit. The web matured as did resources. It was just a few years back when the twitterverse was tied up discussing how the average size of web pages now equaled the size of the [original doom](https://www.wired.com/2016/04/average-webpage-now-size-original-doom/). Many of us mused about what the size the page could become in time, including [our very own Tammy Everts](https://speedcurve.com/blog/web-performance-page-bloat/), but the reality is startling. A page sits @  ~4 MB and 3.7 MB, desktop/mobile respectively, at the 75th percentile, and a shocking 7.4 MB and 6.7 MB at the 90th percentile. There are multitudes of implications in having such heavy pages, like the likelihood of poor user experience due to unreliable networks. Today, despite lessons [learned a decade ago](https://blog.chriszacharias.com/page-weight-matters), we are experiencing variations of the same challenges: despite having slightly better networks, we are working with much larger resources.
+PageWeightStillMatters would almost imply that it didn't or ever mattered. It might not have mattered when text based Craigslist launched. But 25 years ago when it was founded, Mosaic 1.0 also launched the same year, and Waterfalls by TLC was a top hit. The web matured as did resources. It was just a few years back when the twitterverse was tied up discussing how the average size of web pages now equaled the size of the [original doom](https://www.wired.com/2016/04/average-webpage-now-size-original-doom/). Many of us mused about what the size the page could become in time, including [our very own Tammy Everts](https://speedcurve.com/blog/web-performance-page-bloat/), but the reality is startling. A page sits @  ~4 MB and 3.7 MB, desktop/mobile respectively, at the 75th percentile, and a shocking 7.4 MB and 6.7 MB at the 90th percentile. There are multitudes of implications in having such heavy pages, like the likelihood of poor user experience due to unreliable networks. Today, despite lessons [learned a decade ago](https://blog.chriszacharias.com/page-weight-matters), we are experiencing variations of the same challenges: despite having slightly better networks, we are working with much larger resources.
```

Comment:
```suggestion
#PageWeightStillMatters would almost imply that it didn't or ever mattered. It might not have mattered when text based Craigslist launched. But 25 years ago when it was founded, Mosaic 1.0 also launched the same year, and Waterfalls by TLC was a top hit. The web matured as did resources. It was just a few years back when the twitterverse was tied up discussing how the average size of web pages now equaled the size of the [original doom](https://www.wired.com/2016/04/average-webpage-now-size-original-doom/). Many of us mused about what the size the page could become in time, including [our very own Tammy Everts](https://speedcurve.com/blog/web-performance-page-bloat/), but the reality is startling. A page sits @  ~4 MB and 3.7 MB, desktop/mobile respectively, at the 75th percentile, and a shocking 7.4 MB and 6.7 MB at the 90th percentile. There are multitudes of implications in having such heavy pages, like the likelihood of poor user experience due to unreliable networks. Today, despite lessons [learned a decade ago](https://blog.chriszacharias.com/page-weight-matters), we are experiencing variations of the same challenges: despite having slightly better networks, we are working with much larger resources.
```